cibuildwheel/macos.py

```diff
@@ -175,14 +176,14 @@ def build(project_dir, output_dir, test_command, before_test, test_requires, tes
 
         # run the before_build command
         if before_build:
-            before_build_prepared = prepare_command(before_build, project=abs_project_dir)
+            before_build_prepared = prepare_command(before_build, project=abs_project_dir, package=abs_package_dir)
```

Comment:
`project=abs_project_dir` is also not necessary anymore, then, since that should just be the outcome of running `pwd` in the `CIBW_BEFORE_BUILD` command.

Reply:
It would be necessary to keep `project` for backward compatibility, I think, since it might appear in user configs. We could deprecate and remove from the docs, though.

Reply:
Yup, I like that idea! :-)

Reply:
So here I just need to update docs to remove it?